Code/Framework/AzCore/AzCore/std/string/conversions.h

```diff
@@ -384,6 +463,45 @@ namespace AZStd
         return to_wstring(dest, src.c_str(), src.length());
     }
 
+    template<size_t MaxElementCount1>
+    void to_wstring(AZStd::basic_fixed_string<wstring::value_type, MaxElementCount1, wstring::traits_type>& dest, const char* str, size_t strLen = 0)
+    {
+        dest.clear();
+
+        if (strLen == 0)
+        {
+            strLen = strlen(str);
+        }
+
+        if (strLen > 0)
+        {
+            Internal::WCharTPlatformConverter<>::to_wstring(dest, str, str + strLen);
+        }
+    }
+
+    template<size_t MaxElementCount1, size_t MaxElementCount2>
+    void to_wstring(AZStd::basic_fixed_string<wstring::value_type, MaxElementCount1, wstring::traits_type>& dest, const AZStd::basic_fixed_string<string::value_type, MaxElementCount2, string::traits_type>& src)
+    {
+        return to_wstring(dest, src.c_str(), src.length());
+    }
+
+    inline void to_wstring(wchar_t* dest, size_t destSize, const char* str, size_t srcLen = 0)
+    {
+        if (srcLen == 0)
+        {
+            srcLen = strlen(str);
+        }
+
+        if (srcLen > 0)
+        {
+            wchar_t* endWStr = Internal::WCharTPlatformConverter<>::to_wstring(dest, destSize, str, str + srcLen);
+            if (endWStr < (dest + destSize))
+            {
+                *endWStr = '\0'; // null terminator
+            }
+        }
+    }
```

Comment:
question: Are these functions moved or added? Seems like these should really have some accompanying tests (totally in a separate PR but would be good to ensure we have some cases to ensure things are behaving as expected - maybe a follow-up GHI?)

Reply:
they were added, take a look at Code/Framework/AzCore/Tests/AZStd/String.cpp

Reply:
Ah fantastic thanks! Sorry I missed them